docs/usage/dependency-pinning.md

```diff
@@ -142,7 +142,7 @@ You can add a package rule in our Renovate configuration to group these together
 Since both `yarn` and `npm@5` both support lock files, it's a common question to ask "Why should I pin dependencies if I'm already using a lock file?".
 It's a good question!
 
-![broken-lockfile](assets/images/broken-lockfile.jpg)
+![broken-lockfile](assets/images/broken-lockfile.jpg){ loading=lazy }
```

Comment:
I wanted to use the `{ loading=lazy }` directive for the README file as well. But then it looks weird when you view the readme on GitHub.

The image will show normally, but have the text { loading=lazy } right after it.

Maybe we can add the lazy loading directive in the code that pulls in the README as our homepage?